src/scalpel/cfg/builder.py

```diff
@@ -430,6 +431,7 @@ def visit_Try(self, node):
         # Create a new block for the body of try.
         try_block = self.new_block()
         self.add_exit(self.current_block, try_block, ast.Constant(True))
+        print(node.orelse)
```

Comment:
I think this is a dubugging satement

Reply:
Yes, definitely. I am adding a test and then will clean the code 👍 I'll mark the PR as ready.
I cannot run the tests locally so I will put it in `src`.
Also, I couldn't find any tests asserting the resulting CFG. Do you want me to write one for that case or do you prefer not testing this?

Reply:
Please just do whatever you want :)  you can put tests right under the same folder 

I am also thinking to have some unit tests for all control flow statements next (might just be using all the example code in user guide). Currently, the CFG is tested by some other modules.

Reply:
sounds good! :)